src/main/java/software/amazon/awssdk/crt/Log.java

```
@@ -134,8 +134,17 @@ public static void initLoggingFromSystemProperties() throws IllegalArgumentExcep
         String filenameString = System.getProperty(LOG_FILE_NAME_PROPERTY_NAME);
         String levelString = System.getProperty(LOG_LEVEL_PROPERTY_NAME);
 
-        // If nothing was specified, disable logging
-        if (destinationString == null && levelString == null) {
+        initLogging(destinationString, filenameString, levelString);
+    }
+
+    /**
+     * Initializes the logging system
+     * @param destinationString determines where logging should be routed to
+     * @param filenameString filename to use if destination for logs is a file
+     * @param levelString detail level of logs that should be shown
+     */
+    public static void initLogging(String destinationString, String filenameString, String levelString){
+        if(destinationString == null && levelString == null){
             return;
```

We probably should throw exception or return bool when the argument is invalid.